spk/tt-rss/src/bin/settings.sh

```diff
@@ -12,5 +12,152 @@ fi
 
 if [ -z "${INSTALL_DIR}" ]
 then
-    INSTALL_DIR="/usr/local/${PACKAGE}"
+    INSTALL_DIR="/var/packages/${PACKAGE}/target"
 fi
+
+if [ -z "${LOGS_DIR}" ]
+then
+    LOGS_DIR="${INSTALL_DIR}/var/logs"
+fi
+
+if [ -z "${WEB_DIR}" ]
+then
+    WEB_DIR="/var/services/web"
+fi
+
+LOG_FILE="${LOGS_DIR}/${PACKAGE}.log"
+if [ ! -f "${LOG_FILE}" ]
```

Comment:
Force the log file creation and ensure it's created with the appropriate owners and rights.